package.json

```diff
@@ -32,7 +32,7 @@
     "@bcrs-shared-components/date-picker": "1.2.5",
     "@bcrs-shared-components/detail-comment": "1.1.3",
     "@bcrs-shared-components/enums": "1.1.20",
-    "@bcrs-shared-components/fee-summary": "1.2.5",
+    "@bcrs-shared-components/fee-summary": "2.0.40",
```

Comment:
It works fine. I compared the old vs new code and the only difference is where the SBC Fee Summary component came from (was prev imported from sbc-common).

![image](https://github.com/user-attachments/assets/c82ec971-89bd-4daa-b550-1e3828751f7a)